deployment/live/gcp/conformance/README.md

```diff
@@ -30,7 +31,12 @@ You'll need the following tools installed:
 
 #### Google Cloud tooling
 
-Ensure you've got already created a project you want to use, and have configured your local `gcloud`
+[!CAUTION]
+This example creates real Google Cloud resources running in your project. They will almost certainly
+cost you real money if left running.  For the purposes of this demo it is strongly recommended that 
+you create a new project so that you can easily clean up at the end.
```

Comment:
```suggestion
> [!CAUTION]
> This example creates real Google Cloud resources running in your project. They will almost certainly
> cost you real money if left running.  For the purposes of this demo it is strongly recommended that 
> you create a new project so that you can easily clean up at the end.
```

Reply:
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts